src/components/calcite-pick-list-item/calcite-pick-list-item.scss

```diff
@@ -1,13 +1,17 @@
 :host {
-  @extend %component-host;
-  @apply items-stretch
+	@apply items-stretch
     flex
-    text-color-1
+    text-color-link
```

Comment:
This should either be `text-color-1` or not set since we're setting it on title and description.